README.md

```diff
@@ -1,67 +1,61 @@
 # Stylix
 
-Stylix is a NixOS module which applies the same colour scheme, font and
-wallpaper to a range of applications and desktop environments.
+## About
 
-## What's this?
+Stylix is a theming framework for [NixOS](https://nixos.org), [Home
+Manager](https://github.com/nix-community/home-manager), and
```

Comment:
Would <https://nix-community.github.io/home-manager/#ch-introduction> be more friendly for new users? It has more explanation of what Home Manager does, whereas the README is mostly usage instructions.